.custom_wordlist.txt

```diff
@@ -26,17 +30,24 @@ aws
 backend
 backoff
 balancers
+bootable
 BSL
 BuildKit
 bzip
 Center
 CentOS
+changelog
+Changelog
```

Comment:
You just need to add 'changelog' and 'changelogs' here. The spellchecker will automatically consider capitalised versions to be correct. So they can be removed from here.

Reply:
Ahh, good to know. I was bulk adding based on the `make spelling` results.